mappings/net/minecraft/util/shape/AbstractVoxelShapeContainer.mapping

```diff
@@ -0,0 +1,30 @@
+CLASS cmm net/minecraft/util/shape/AbstractVoxelShapeContainer
+	CLASS cmm$b
+	FIELD a xSize I
+	FIELD b ySize I
+	FIELD c zSize I
+	FIELD d AXES [Ley$a;
+	METHOD <init> (III)V
+		ARG 1 xSize
+		ARG 2 ySize
+		ARG 3 zSize
+	METHOD a isEmpty ()Z
+	METHOD a modify (IIIZZ)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 resize
+		ARG 5 included
+	METHOD a getMin (Ley$a;)I
+		ARG 1 axis
+	METHOD b getXSize ()I
+	METHOD b contains (III)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD b getMax (Ley$a;)I
+		ARG 1 axis
+	METHOD c getYSize ()I
+	METHOD c getAxisSize (Ley$a;)I
```

Comment:
if we have getMin and getMax we can prob also have getSize... or maybe getLength? Min... Max; is Size a good term?

Reply:
getSize is better, IMO. I'd expect length to be used in conjunction with width & height, but it's not here.

Reply:
Right. IMO don't use width/height in 3D space; it's a tad confusing.

Reply:
Fixed.